stablehlo/dialect/StablehloOps.cpp

```diff
@@ -807,6 +819,14 @@ LogicalResult AbsOp::inferReturnTypes(
   return hlo::inferAbsOp(location, adaptor.getOperand(), inferredReturnTypes);
 }
 
+//===----------------------------------------------------------------------===//
+// CollectiveBroadcastOp
+//===----------------------------------------------------------------------===//
+
```

Comment:
Strange, `CollectivePermuteOp` seems to be the only op that splits them. Lmk if you found other ops like this; it would be nice to sync. If so, I'm supportive of fixing them altogether in another PR. Thanks for pointing this out!

Reply:
That seems fair to me. I'll fix CollectivePermuteOp in this PR too then.

Reply:
Thanks!